test/core/TakerOperations.t.sol

```diff
@@ -756,13 +758,13 @@ contract TakerOperationsTest is MangroveTest {
   }
 
   function test_unsafe_gas_left_fails_order() public {
-    mgv.setGasbase($(base), $(quote), 1);
     quote.approve($(mgv), 1 ether);
     uint ofr = mkr.newOfferByVolume(1 ether, 1 ether, 120_000, 0);
+    mgv.setGasbase($(base), $(quote), 1);
     Tick offerTick = pair.offers(ofr).tick();
 
     vm.expectRevert("mgv/notEnoughGasForMakerTrade");
```

Comment:
How can that revert still happen? I think I'm missing something

Reply:
Because I set gasbase to 1 which breaks the accounting by not setting aside enough at the beginning of the market order.

Reply:
ahh of course :)